packages/gatsby-cli/src/__tests__/handlers/plugin-add.ts

```diff
@@ -0,0 +1,121 @@
+import { copyFile, readFile, rm } from "fs/promises"
```

Comment:
Normally if the platform has something I prefer to use that, no reason to increase our reliance on a third party dependency for no gain.

Looks like we're pretty married to `fs-extra` already though so I'll keep this in mind

Reply:
Major reason is because of: https://github.com/jprichardson/node-fs-extra

> It also uses [graceful-fs](https://github.com/isaacs/node-graceful-fs) to prevent EMFILE errors.

This handles some stuff that normal fs doesn't

Reply:
That's a good reason, will update it